client/src/main/java/io/pravega/client/stream/StreamCut.java

```diff
@@ -24,12 +25,42 @@
      * This is used represents an unbounded StreamCut. This is used when the user wants to refer to the current HEAD
      * of the stream or the current TAIL of the stream.
      */
-    StreamCut UNBOUNDED = () -> null;
+    StreamCut UNBOUNDED = new StreamCut() {
```

Comment:
This looks like a conflict with PR #2507.

Reply:
yes, it is.